src/odemis/gui/cont/tabs.py

```diff
@@ -382,6 +381,11 @@ def __init__(self, name, button, panel, main_frame, main_data):
         panel.vp_secom_tl.canvas.remove_view_overlay(panel.vp_secom_tl.canvas.play_overlay)
         panel.vp_secom_tr.canvas.remove_view_overlay(panel.vp_secom_tr.canvas.play_overlay)
 
+        # Add a sample overlay to each view (if we have information)
+        if main_data.sample_centers:
```

Comment:
```suggestion
        if main_data.sample_centers and main_data.sample_radius:
```
maybe a better check